app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer.go

```diff
@@ -275,12 +297,21 @@ func formatLoadBalancersList(balancers []status.LoadBalancerStatus) (string, err
 }
 
 func (l *LoadBalancerSyncer) ListLoadBalancers() (string, error) {
-	balancers, err := l.frs.ListLoadBalancerStatuses()
-	if err != nil {
-		return "", err
+	// We fetch the list from both url maps and forwarding rules and then merge them.
+	// This is because we first used to store the status on forwarding rules and then migrated to url maps.
+	// https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/issues/145 has more details.
+	umBalancers, umErr := l.ums.ListLoadBalancerStatuses()
+	if umErr != nil {
```

Comment:
Is it appropriate to return on any error? Or should you continue for some errors because you are merging?

Reply:
ListLoadBalancerStatus returns an error only if its a real error and cant be ignored. If status is not found on forwarding rule, for ex, then it does not return an error. It returns status for MCIs that have their status on forwarding rules.

Updated to merge the errors from both url map and forwarding rule and return both at once

Reply:
Cool, ok.